lib/feed.sh

```diff
@@ -53,24 +54,15 @@ readSource() {
 	verbose --list "readSource" "src=$_src" "${_assetPairs[@]}"
 
 	case "$_src" in
-		setzer)
+		setzer|gofer)
 			for _assetPair in "${_assetPairs[@]}"; do
 				log "Querying price and calculating median" "source=$_src" "asset=${_assetPair}"
-#				readSourcesWithSetzer "$_assetPair"
-#				2> >(STDERR_DATA="$(cat)"; [[ -z "$STDERR_DATA" ]] || error "source-setzer [stderr]" "$STDERR_DATA") \
-				"source-setzer" "$_assetPair" \
-				| tee >(_data="$(cat)"; [[ -z "$_data" ]] || verbose --raw "source-setzer" "$(jq -sc <<<"$_data")") \
-				|| error "Failed to get price" "app=source-setzer" "asset=$_assetPair"
+
+				"source-$_src" "$_assetPair" \
+				| tee >(_data="$(cat)"; [[ -z "$_data" ]] || verbose --raw "source-$_src" "$(jq -sc <<<"$_data")") \
+				|| error "Failed to get price" "app=source-$_src" "asset=$_assetPair"
 			done
 			;;
-		gofer)
```

Comment:
For loop for setzer, single call for gofer. This change will result in a behavior change

Reply:
yes, but gofer has a caching mechanism so we are not loosing any efficiency and we are gaining simplicity